setup-env/action.yaml

```diff
@@ -30,5 +30,7 @@ runs:
     - name: Checkout code
       uses: actions/checkout@v3
       with:
+        fetch-depth: 0
         submodules: true
         token: ${{ inputs.token }}
+        clean: false
```

Comment:
What is clean: false for?

Reply:
> The clean option determines whether the repository should be cleaned before checking out the code. When set to false, it means that the repository will not be cleaned, and any existing files or changes in the repository will be preserved.

By default, the clean option is set to true, which performs a clean checkout by removing any existing files or changes in the repository before checking out the code.

In this particular case, setting clean to false ensures that any existing files or changes in the repository are retained during the checkout process. This can be useful if you want to keep the state of the repository intact and avoid losing any modifications or files that may already be present.